src/services/item/plugins/file/index.ts

```diff
@@ -45,9 +45,8 @@ const basePlugin: FastifyPluginAsync<GraaspPluginFileOptions> = async (fastify,
     },
   });
 
-  // if (!buildFilePath) {
-  //   throw new Error('graasp-plugin-file: buildFilePath is not defined');
-  // }
+  // "install" custom schema for validating document items update
+  extendExtrasUpdateSchema(updateSchema(fileService.type));
```

Comment:
Does this do anything? I only find lambdas returning schemas (`updateOne` in items fluent schema)
What does this do written alone like this without being assigned somewhere?

Reply:
It's a dirty legacy trick: `extendExtrasUpdateSchema` as a local yet saved schema that is extended given new schemas. 

Later in `PATCH /items/id` it reuses this saved schema that got updated during the app setup (that goes through all the plugins). It allows to edit specific fields that only exists depending on the item type.